.maintain/gitlab/check_polkadot_companion_build.sh

```diff
@@ -57,9 +57,9 @@ then
   # get the last reference to a pr in polkadot
   pr_data="$(curl -sSL -H "${github_header}" -s ${github_api_substrate_pull_url}/${CI_COMMIT_REF_NAME})"
   pr_ref="$(echo $pr_data | grep -Po '"ref"\s*:\s*"\K(?!master)[^"]*')"
-  pr_body="$(echo $pr_data | sed -n -r 's/^[[:space:]]+"body": (".*")[^"]+$/\1/p')"
+  pr_body="$(echo "$pr_data" | sed -n -r 's/^[[:space:]]+"body": (".*")[^"]+$/\1/p')"
 
-  pr_companion="$(echo "${pr_body}" | sed -n -r \
+  pr_companion="$(echo "${pr_data}" | sed -n -r \
```

Comment:
this would make the pr_body variable useless and would have to be adapted by the next step then as well.

Reply:
oh yes, I am removing it (I was not sure about the second use).